frontend/src/metabase/visualizations/lib/settings/graph.ts

```diff
@@ -492,52 +510,58 @@ export const GRAPH_DISPLAY_VALUES_SETTINGS = {
   "graph.other_category_aggregation_fn": {
     hidden: true,
     getDefault: ([{ data }], settings) => {
-      const [metricName] = settings["graph.metrics"];
+      const [metricName] = settings["graph.metrics"] ?? [];
```

Comment:
well... since the previous version works, it doesn't look like `graph.metrics` could be non-array to me.

By doing this, we say `graph.metrics` could be falsy. Which could be a misdirection.